File: Programs/C/prime.c
/*

Program to print prime number

*/

#include<stdio.h>
main() {
   int i,n,p,c;

   printf("Enter any number:");
   scanf("%d",&n);

   c = 2;

   while( c <= n) {

      i = 2;
      p = 1;

   while (i < c && p == 1) {  

      if( c % i == 0) {

           p = 0;
     }
      i++;
   }
         
     if(p == 1) {

       printf(" %d, ",c);
     }
      c++;
   }
      printf("\n");
}

File: Programs/Java/Prime.java
import java.util.Scanner ;

class Prime {


public static void main( String[] args ) {

   Scanner sc = new Scanner ( System.in ) ;

   System.out.print("Enter any value : ") ;
   int n = sc.nextInt() ; 

    int d = 2 ;
    int q = 0 ;

   while( d < n ) {

      if( n % d == 0 ) { q = 1 ; }
      d++ ;
   }   


   if( q == 0 ) {

      System.out.println("The given number is prime." ) ;
   }

   else { 

      System.out.println("Not prime." ) ;
   }

}


}
